pyproject.toml

```diff
@@ -6,7 +6,7 @@ build-backend = 'setuptools.build_meta'
 name = 'vivisect'
 version = '1.1.1'
 authors = [
-    { name =  'The Vivisect Peeps', email = ''}
+    { name =  'The Vivisect Peeps', email = 'vivdevs@r4780y.com'}
```

Comment:
this can be whatever you like, @rakuy0

Reply:
As long as it forwards to the two of us, I'm cool with whatever.